### PR TITLE
f/android_instructions

### PR DESCRIPTION
**Pull request by mspensieri**
Explain minimum android sdk version, and add warning to eclipse instructions

@jugarrit @dannytranlx @alavers @spasiu @Mario54

### CODE REVIEW

File: source/includes/_adding_smooch_android.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # Adding Smooch to your app
 
-There are a few ways you can include the Smooch plugin on your Android project.
+The Smooch library is distrubuted in both AAR and JAR format. If you are using Android Studio, follow the instructions for [installation of the AAR package](#android-studio-with-gradle). Users of Eclipse or Netbeans must follow the [installation instructions for the JAR package](#eclipse-netbeans-and-the-ant-build-system).
```
Comment (Mario54) [inline source/includes/_adding_smooch_android.md]: Typo `distrubuted`

Comment (mspensieri) [inline source/includes/_adding_smooch_android.md]: Fixed

**Comment by Mario54** [comment]
:+1:

File: source/includes/_adding_smooch_android.md [hunk 2]
```diff
@@ -35,6 +36,8 @@ Smooch depends on a few external libraries that many applications already featur
 
 ## Eclipse, Netbeans and the ant build system
 
+<aside class="warning">This section only applies if you are using Eclipse or Netbeans. If you are using Android Studio and have already followed the [installation instructions](#android-studio-with-gradle), you can skip straight to the [initialization step](#initialize-smooch-in-your-app).</aside>
```
Comment (dannytranlx) [inline source/includes/_adding_smooch_android.md]: Well played!

**Comment by dannytranlx** [comment]
:+1:

**Pull request merged**